packages/core/src/App/Containers/Modals/crypto-transaction-processing-modal/crypto-transaction-processing-modal.scss

```diff
@@ -0,0 +1,60 @@
+.crypto-transaction-processing-modal {
+    &__content {
+        height: 100%;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        gap: 2.4rem;
+        padding: 2.4rem;
+
+        @include mobile-or-tablet-screen {
+            position: absolute;
+            top: 50vh;
+            transform: translateY(-50%);
+            padding: 1.6rem;
+        }
+
+        &-description {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+        }
+    }
+
+    &__circle {
+        position: relative;
+        width: 64px;
+        height: 64px;
+        background: $color-yellow;
+        border-radius: 32px;
+    }
+
+    &__dot {
+        position: absolute;
+        width: 10px;
+        height: 10px;
+        background: var(--general-main-1);
+        border-radius: 5px;
+
+        &-0 {
+            top: 27px;
```

Comment:
All the top is same, may be we can just modify the left?

Reply:
@habib-deriv In my opinion coordinates should be part of every dot, becuase it's related to their position, but ok, I'll change it.